src/player/player.gd

```diff
@@ -4,7 +4,7 @@ extends Area2D
 @onready var boots_packed_scene: PackedScene = preload("res://src/boots/boots.tscn")
 @onready var raycast := $RayCast2D
 
-signal walked_into_stairs
+signal walked_into_stairs(toward_which_room_id)
```

Comment:
avoid "which" and other question-asking words in parameters/variables, because variables don't ask questions, they're just data. functions use variables' data to ask questions.

Reply:
Great point, will correct in next pr